api/sk-serve/graph/team.resolvers.go

```diff
@@ -40,6 +40,12 @@ func (r *Resolver) Team() generated.TeamResolver { return &teamResolver{r} }
 type deadCapResolver struct{ *Resolver }
 type teamResolver struct{ *Resolver }
 
+// !!! WARNING !!!
```

Comment:
I fix this in my branch, so once that is merged in you'll need to pull that in here